tedana/selection/select_comps.py

```diff
@@ -188,87 +223,99 @@ def selcomps(seldict, comptable, mmix, manacc, n_echos):
     f. Estimate a low and high variance
     """
     # Step 2a
+    # Upper limit for variance explained is median across components with high
+    # Kappa values. High Kappa is defined as Kappa above Kappa elbow.
     varex_upper_p = np.median(
         comptable.loc[comptable['kappa'] > getelbow(comptable['kappa'], return_val=True),
                       'variance explained'])
     ncls = acc.copy()
     # NOTE: We're not sure why this is done, nor why it's specifically done
     # three times. Need to look into this deeper, esp. to make sure the 3
     # isn't a hard-coded reference to the number of echoes.
+    # Reduce components to investigate as "good" to ones in which change in
+    # variance explained is less than the limit defined above.... What?
     for nn in range(3):
         ncls = comptable.loc[ncls].loc[
             comptable.loc[
                 ncls, 'variance explained'].diff() < varex_upper_p].index.values
 
-    # Compute elbows
-    kappas_lim = comptable.loc[comptable['kappa'] < utils.getfbounds(n_echos)[-1], 'kappa']
-    kappa_elbow = np.min((getelbow(kappas_lim, return_val=True),
+    # Compute elbows from other elbows
+    kappas_under_f01 = (comptable.loc[comptable['kappa'] <
+                        utils.getfbounds(n_echos)[-1], 'kappa'])
+    # NOTE: Would an elbow from all Kappa values *ever* be lower than one from
+    # a subset of lower values?
+    kappa_elbow = np.min((getelbow(kappas_under_f01, return_val=True),
                           getelbow(comptable['kappa'], return_val=True)))
     rho_elbow = np.mean((getelbow(comptable.loc[ncls, 'rho'], return_val=True),
                          getelbow(comptable['rho'], return_val=True),
                          utils.getfbounds(n_echos)[0]))
 
-    # Initial guess of good components based on Kappa and Rho elbows
-    good_guess = ncls[(comptable.loc[ncls, 'kappa'] >= kappa_elbow) &
-                      (comptable.loc[ncls, 'rho'] < rho_elbow)]
+    # Initial guess of accepted components based on Kappa and Rho elbows
+    acc_poss = ncls[(comptable.loc[ncls, 'kappa'] >= kappa_elbow) &
```

Comment:
Is the `poss` in this variable name short for `possible` ? I stumbled on this at first. WDYT about `acc_prov` for `provisionally accepted` ? Either way, can we update the comment immediately above to spell this out ?

Reply:
That's totally cool by me.